src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (!sourceString.trim()) {
+    return {};
+  }
+
+  return sourceString
+    .split(';')
+    .map((keyVal) => keyVal.split(':').map((str) => str.trim()))
+    .filter(([key, value]) => key && value)
+    .reduce((accumulator, [key, value]) => {
+      accumulator[key] = value;
+
+      return accumulator;
+    }, {});
 }
```

Comment:
The function `convertToObject` is well-implemented using string methods and array iteration methods as recommended. However, it's important to note that the code does not handle cases with multiple colons within the style string's property values or multiple semicolons without any styles between them. While this is not required for the task, it would be a nice addition to make the function more robust.